### PR TITLE
change job offer style and fix redirect path

### PR DESCRIPTION
**Pull request by nioushamahjoub**
change job offer style
fix redirect path
#28

### CODE REVIEW

File: src/main/java/ca/concordia/soen6011/controller/AddJobOfferController.java [hunk 1]
```diff
@@ -44,13 +44,13 @@ public String saveJobOffer(@RequestParam("title") String title
             , @RequestParam("highsalery") int highsalery) {
 
         Employer employer = new Employer();
-        employer.setId(1);
+        employer.setId(7);
 
         Joboffers joboffers = new Joboffers(title, description, staff, company, published,  employer,
                 highsalery,lowsalery, deadline, JobOfferStatus.PENDING);
 
         jobOfferRepository.save(joboffers);
```
Comment (sepehr9877) [inline src/main/java/ca/concordia/soen6011/controller/AddJobOfferController.java:52]: 👍

**Review by sepehr9877 — APPROVED** [review]

File: src/main/java/ca/concordia/soen6011/controller/AddJobOfferController.java [hunk 2]
```diff
@@ -44,13 +44,13 @@ public String saveJobOffer(@RequestParam("title") String title
             , @RequestParam("highsalery") int highsalery) {
 
         Employer employer = new Employer();
-        employer.setId(1);
+        employer.setId(7);
```
Comment (SoroorAghamiri) [inline src/main/java/ca/concordia/soen6011/controller/AddJobOfferController.java:47]: You can find the Id with the cached username now. Do you want to update this line?

File: src/main/java/ca/concordia/soen6011/controller/AddJobOfferController.java [hunk 3]
```diff
@@ -44,13 +44,13 @@ public String saveJobOffer(@RequestParam("title") String title
...
 
         Joboffers joboffers = new Joboffers(title, description, staff, company, published,  employer,
                 highsalery,lowsalery, deadline, JobOfferStatus.PENDING);
 
         jobOfferRepository.save(joboffers);
-        return "redirect:/addJobOffer";
+        return "redirect:/employer/myJobOffers";
```
Comment (SoroorAghamiri) [inline src/main/java/ca/concordia/soen6011/controller/AddJobOfferController.java:53]: I think myJobOffers should be accessed through /profile/{username}/myJobOffers. @MajorTreble is that correct?

Comment (nioushamahjoub) [reply src/main/java/ca/concordia/soen6011/controller/AddJobOfferController.java:53]: the username is not in this file yet

**Review by SoroorAghamiri — APPROVED** [review]

**Pull request merged**